src/lib/plugin.js

```diff
@@ -80,25 +79,10 @@ module.exports = class PluginPaymentChannel extends EventEmitter2 {
     this._requestHandler = null
     this._sideProtoHandler = {}
 
-    if (opts.server) {
-      assertOptionType(opts, 'server', 'string')
-
-      // token can be provided via the URL
-      // e.g. btp+wss://:token@host.example:1234/path
-      const parsed = url.parse(opts.server)
-      if (parsed.auth) {
-        const [, token] = parsed.auth.split(':')
-        opts.token = token
-
-        // remove token from url
-        parsed.auth = null
-        opts.server = url.format(parsed)
-      }
-
+    if (opts.btpUri) {
+      assertOptionType(opts, 'btpUri', 'string')
       this._client = new BtpClient({
-        server: opts.server,
-        secret: opts.secret,
```

Comment:
we should still allow the token to be passed in separately. There may be cases (e.g. in a UI) where the server URI is shown but not the secret. If we don't allow the token to be passed in separately, the UI would have to do string manipulation to pass in authentication information. Just like in HTTP basic auth, the URI isn't the only way to tell your browser to send those credentials.

Reply:
see [my comment above](https://github.com/interledgerjs/ilp-plugin-payment-channel-framework/pull/27)